client/my-sites/backup/wpcom-backup-upsell.tsx

```diff
@@ -50,17 +51,11 @@ const BackupVPActiveBody: FunctionComponent = () => {
 	const onUpgradeClick = useTrackCallback( undefined, 'calypso_jetpack_backup_vaultpress_click' );
 	return (
 		<PromoCard
-			title={ preventWidows( translate( 'Your site has VaultPress' ) ) }
-			image={ { path: JetpackBackupErrorSVG } }
+			title={ preventWidows( translate( 'Your backups are powered by VaultPress' ) ) }
```

Comment:
Love your attention to details using `preventWidows`!

Reply:
I can't claim credit since I only changed the text -- the function call was already there -- but I think it's a pretty neat function too! Definitely a TIL moment for me.

Reply:
^ +1!